.github/pull_request_template.md

```diff
@@ -37,19 +37,19 @@ _Optional. Remove if not applicable._
 
 ## Documentation
 
-#### Requires documentation change?
+#### Requires documentation update
 
 - [ ] Yes
 - [ ] No
 
-_Link to documentation or issue about change which needs to be added with "docs" label (example: https://github.com/NethermindEth/nethermind/issues/5856)_
+_If yes, fill in the details here or link the PR to docs update. Remove if not applicable._
```

Comment:
@rubo 
Thing with my "issue" approach was as discussed on call to create a separate issue with "docs" label and put it in appropriate project later to not forgot about this.
If you will just push details here and close PR, then those changes for sure will be forgotten.

Reply:
@rubo any opinion on above?

Reply:
What about #5868?